src/state/quick-bar-mixin.ts

```diff
@@ -48,8 +48,15 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
     }
 
     private _inInputField(e: KeyboardEvent) {
-      return ["INPUT", "TEXTAREA"].includes(
-        (e.composedPath()[0] as HTMLElement).tagName
+      const el = e.composedPath()[0] as any;
+      return (
+        el.tagName === "TEXTAREA" ||
+        (el.tagName === "INPUT" &&
+          ["TEXT", "NUMBER"].includes(this._getInputType(el)))
```

Comment:
It would not be just text and number, but also email, date, search, color, tel, time, email, URL, etc.

Reply:
Maybe easier to check the types we do want the event to handle.

Reply:
Done